Cliqz/Privacy/UI/ControlCenter/TrackersController.swift

```diff
@@ -298,7 +298,7 @@ extension TrackersController: UITableViewDataSource, UITableViewDelegate {
 		cell.infoButtonAction = { [weak self] (button, trackerName) in
 			var url = "https://whotracks.me/tracker-not-found.html"
 			if let t = trackerName {
-				if let d = Engine.sharedInstance.getBridge().callAction("mobile-cards:getTrackerDetails", args: [t.lowercased()]) as? [String: Any],
+				if let d = Engine.sharedInstance.getBridge().callAction("insights:getTrackerDetails", args: [t.lowercased()]) as? [String: Any],
```

Comment:
`mobile-cards:getTrackerDetails` will not work for sure, so good that you removed it. But `insights:getTrackerDetails` will not work as well, since I don't see any relevant code in the extension side. Let's have a look tomorrow together, I will explain how we do it in the extension.

Reply:
https://cliqztix.atlassian.net/browse/IP-532
check this out, I just followed to the instructions

Reply:
Ok looks fine! We can merge!